test/models/comment_test.rb

```diff
@@ -1,7 +1,35 @@
 require 'test_helper'
 
 class CommentTest < ActiveSupport::TestCase
-  # test "the truth" do
-  #   assert true
-  # end
+  test 'user_id must be present' do
+    comment = Comment.new(user_id: nil, post_id: 1, content: 'Test comment')
+    assert_not comment.valid?
+    assert_includes comment.errors[:user_id], 'can\'t be blank'
+  end
+
+  test 'post_id must be present' do
+    comment = Comment.new(user_id: 1, post_id: nil, content: 'Test comment')
+    assert_not comment.valid?
+    assert_includes comment.errors[:post_id], 'can\'t be blank'
+  end
+
+  test 'content must not be blank' do
+    comment = Comment.new(user_id: 1, post_id: 1, content: '')
+    assert_not comment.valid?
+    assert_includes comment.errors[:content], 'can\'t be blank'
+  end
+  # rubocop:disable Lint/UselessAssignment
+  test 'user_id and post_id combination should be unique' do
+    existing_comment = Comment.create(user_id: 1, post_id: 1, content: 'Test comment 1')
+    # rubocop:enable Lint/UselessAssignment
+    duplicate_comment = Comment.new(user_id: 1, post_id: 1, content: 'Test comment 2')
+    assert_not duplicate_comment.valid?
+    assert_includes duplicate_comment.errors[:user_id], 'has already been taken'
+  end
+
```

Comment:
![image](https://github.com/Kidd254/BlogApp/assets/93039359/15ac1929-dbeb-4aa2-9662-10dc9b82a348)

**[OPTIONAL BUT HIGHLY RECOMMENDED]**

- [ ] It would be nice if you remove this migration which is causing an issue :fearful: because author_id already exists in the posts table so there is not need of this migration :blush: